lua/octo/workflow_runs.lua

```diff
@@ -307,10 +305,11 @@ local function get_logs(id)
     end
 
     local sanitized_name = node.id:gsub("/", ""):gsub(":", ""):gsub(">", "")
-    --Make more than 3 consecutive dots at the end of line into ...
+    --Make more than 3 consecutive dots at the end of line into */. This avoids a bug with unreliable filename endings
     local sanitized_job_id = node.job_id:gsub("/", ""):gsub(":", ""):gsub("%.+$", "*/")
     local file_name = string.format("%s_%s.txt", node.number, sanitized_name)
-    local path = sanitized_job_id .. file_name
+    local path = vim.fs.joinpath(sanitized_job_id, file_name)
```

Comment:
This fixes a bug where it would fail to extract the correct file if there were no `/` at the end of the directory name